packages/content-source-maps/src/utils.ts

```diff
@@ -15,3 +17,25 @@ export function clone<T extends Record<string, unknown> | Array<unknown>>(incomi
     return incoming;
   }
 }
+
+// We only support these editor interfaces: singleLine, tagEditor, listInput, checkbox, richTextEditor, multipleLine, markdown
+// We have to filter out the unsupported editor interfaces, instead of filtering the supported ones, because customers can add custom editor interfaces that can have any name.
+export const UNSUPPORTED_EDITOR_INTERFACES: EditorInterface[] = [
```

Comment:
Maybe it's better to do this the other way around

`SUPPORTED_EDITOR_INTERFACES` 

just to account for new editors types being added in future

Reply:
Yes I had this first but commented above it why we unfortunately can't do this
Editor interfaces can be custom apps from users, which means they can have any string value (which we do want to support)

Reply:
Sorry I'm confused can you not just do the opposite filtering? also are there not custom editors we don't want to support also?

Reply:
Now we have the info if the widget is builtin or a custom one, so I changed the logic to flip the filtering 👍